README.md

```diff
@@ -10,34 +10,30 @@
 [![Join the chat at https://gitter.im/dequelabs/axe-core](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/dequelabs/axe-core?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Package Quality](http://npm.packagequality.com/shield/axe-core.svg)](http://packagequality.com/#?package=axe-core)
 
-The Accessibility Engine for automated testing of HTML-based user interfaces. Drop the axe on your accessibility defects!
+![Axe Logo](doc/axelogo2018.png)
 
-Thanks for helping us to make axe-core the most widely used HTML accessibility testing library in the galaxy! Share your axe-core story using #axecoresquad or by submitting it [here](https://accessibility.deque.com/axe-core-stories) to get a sweet T-shirt.
+Axe is an accessibility testing engine for websites and other HTML-based user interfaces. It's fast, secure, lightweight, and was built to seamlessly integrate with any existing test environment so you can automate accessibility testing alongside your regular functional testing.
```

Comment:
"HTML-based interfaces" will be outdated soon, should we make a note to update it later? cc @caitlinthefirst

Reply:
Yeah, I think that would be a good idea. We'll need to chat about what to put there instead.

Reply:
Axe-core will always be "HTML-based", even if we include linting, it'll never be for languages that do not target HTML.